apps/admin-portal/src/components/applications/attribute-management/attribute-settings.tsx

```diff
@@ -500,18 +500,19 @@ export const AttributeSettings: FunctionComponent<AttributeSelectionPropsInterfa
         findLocalClaimDialectURI();
     }, []);
 
+    // Set the dialects for inbound protocols
     useEffect(() => {
         if (isEmpty(dialect)) {
             return
         }
         //TODO  move this logic to backend
         setIsClaimRequestLoading(true);
-        if (selectedInboundProtocol?.id === SupportedAuthProtocolTypes.SAML) {
+        if (selectedInboundProtocol?.id === SupportedAuthProtocolTypes.OIDC) {
             setIsClaimRequestLoading(false);
-            changeDialect(localDialectURI);
-        } else if (selectedInboundProtocol?.id === SupportedAuthProtocolTypes.OIDC) {
+            changeSelectedDialect("http://wso2.org/oidc/claim")
```

Comment:
Wouldn't it be better if we move `http://wso2.org/oidc/claim` to the constants file?

Reply:
Rather than constant file if we can get this from backends meta or runtime config file it would be great.